packages/fiori/src/ViewSettingsDialog.js

```diff
@@ -667,6 +667,56 @@ class ViewSettingsDialog extends UI5Element {
 		// Invalidate
 		this._currentSettings = JSON.parse(JSON.stringify(this._currentSettings));
 	}
+
+	/**
+	 * Sets a JavaScript object, as settings to the ui5-view-settings-dialog.
+	 * This method can be used after the dialog is initially open, as the dialog need to set it's initial settings.
```

Comment:
* This method can be used after the dialog is initially open, as the dialog needs to set its initial settings.